lib/child_process.js

```diff
@@ -841,17 +864,20 @@ function checkExecSyncError(ret, args, cmd) {
  *   }} [options]
  * @returns {Buffer | string}
  */
-function execFileSync(command, args, options) {
-  options = normalizeSpawnArguments(command, args, options);
+function execFileSync(file, args, options) {
+  const opts = normalizeExecFileArgs(file, args, options);
+
+  file = opts.file;
+  args = opts.args;
+  options = opts.options;
```

Comment:
Is `opts` necessary?
```suggestion
  ({ file, args, options } = normalizeExecFileArgs(file, args, options));
```

Reply:
Thanks, I didn't know this syntax in the past